pkg/gcp/gcp.go

```diff
@@ -27,20 +27,30 @@ import (
 )
 
 type gcpCloud struct {
-	CloudInfo
+	*CloudInfo
 }
 
 // NewCloud creates a new api.Cloud instance which can prepare GCP for Submariner to be deployed on it.
-func NewCloud(info CloudInfo) api.Cloud {
-	return &gcpCloud{CloudInfo: info}
+func NewCloud(info *CloudInfo) api.Cloud {
```

Comment:
I assume you changed this to a pointer to avoid a linter error re: the struct size but it's preferable to pass by value here since `info` is mutated. Ignore the linter error with `//nolint`...

Reply:
Done,